cypress/e2e/default/learn/common-components/navbar.ts

```diff
@@ -0,0 +1,85 @@
+const navBarselectors = {
+  heading: "[data-test-label='landing-header']",
+  smallCallToAction: "[data-test-label='landing-small-cta']",
+  navigationLinks: '.nav-list',
+  avatarContainer: '.avatar-container',
+  defaultAvatar: '.avatar-container',
+  menuButton: '.toggle-button-nav',
+  avatarImage: '.avatar-container .avatar'
+};
+
+
+
+
+describe('Navbar when logged out', () => {
+  beforeEach(() => {
+    cy.visit('/').then(() => cy.viewport(1200, 660));
+  });
+
+  it('Should have a "Sign in" button', () => {
+    cy.contains("[data-test-label='landing-small-cta']", 'Sign in');
+  });
+
+  it(
+    'Should have `Sign in` link on landing and learn pages' +
+    ' when not signed in',
+    () => {
+      cy.contains(navBarselectors.smallCallToAction, 'Sign in');
+      cy.get(navBarselectors.menuButton).click();
+      cy.get(navBarselectors.navigationLinks).contains('Curriculum').click();
+      cy.contains(navBarselectors.smallCallToAction, 'Sign in');
+    }
+  );
+});
+
+describe('Navbar Logged in', () => {
+  beforeEach(() => {
+    cy.visit('/').then(() => cy.login()).then(() => cy.viewport(1200, 660));
+  });
+
+  it('Should render properly', () => {
+    cy.get('#universal-nav').should('be.visible');
+    cy.get('#universal-nav').should('have.class', 'universal-nav');
+  });
+
+  it(
+    'Should take user to learn page when clicked on ' + 'the freeCodeCamp logo',
+    () => {
+      cy.get('.universal-nav-middle').within(() => {
+        cy.get('svg').click();
+      });
+      cy.url().should('include', '/learn');
+    }
+  );
+
+  // have the curriculum and CTA on landing and /learn pages.
+  it(
+    'Should have `Radio`, `Forum`, and `Curriculum` links on landing and learn pages' +
+    'page when not signed in',
+    () => {
+      cy.get(navBarselectors.menuButton).click();
+      cy.get(navBarselectors.navigationLinks).contains('Forum');
+      cy.get(navBarselectors.navigationLinks).contains('Curriculum').click();
+      cy.url().should('include', '/learn');
+      cy.get(navBarselectors.navigationLinks).contains('Curriculum');
+      cy.get(navBarselectors.navigationLinks).contains('Forum');
+      cy.get(navBarselectors.navigationLinks).contains('Radio');
+    }
+  );
+
+  it('Should have `Profile` link when user is signed in', () => {
+    cy.get(navBarselectors.menuButton).click();
+    cy.get(navBarselectors.navigationLinks).contains('Profile').click();
+    cy.url().should('include', '/developmentuser');
+  });
+
+  it('Should have a profile image with class `default-border`', () => {
+    cy.get(navBarselectors.avatarContainer).should('have.class', 'default-border');
```

Comment:
```suggestion
    cy.get(navBarselectors.avatarContainer).should(
      'have.class',
      'default-border'
    );
```

Reply:
That step was completed two days ago but I never managed to push it.